core/src/test/scala/org/bykn/bosatsu/TestUtils.scala

```diff
@@ -155,18 +156,33 @@ object TestUtils {
       val normPackMap = NormalizePackageMap(infPackMap).normalizePackageMap
       (for {
         pack <- normPackMap.toMap.get(mainPack)
-        (name, rec, expr) <- pack.program.lets.lastOption
+        exprs <- pack.program.lets.traverse {case (_, rec, expr) =>
+          expr.traverse[Option, (Declaration, Normalization.NormalExpressionTag, String)]{
+            case (d, nt) => Some((d, nt, nt.ne.serialize(x => x)))
+          }
+        }
+        expr <- exprs.lastOption
```

Comment:
this is a bit confusing because `expr` is shadowed below. Can we move this below fleft and fright?

Reply:
sure. I can also rename `expr` above to just `e`